src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,76 @@
+import usersFromServer from '../../api/users';
+
+const TITLE_ERROR_MSG = 'Please enter a title';
+const USER_ERROR_MSG = 'Please choose a user';
+
+interface Props {
+  handleSubmit: (event: React.FormEvent) => void;
+  handleTitleInput: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  handleUserSelect: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+
+  formInputs: {
+    title: string;
+    userId: number;
+  };
+  firstOptionDisabled: boolean;
+  errorStatuses: {
+    title: boolean,
+    user: boolean,
+  };
```

Comment:
For `formInputs` and `errorStatuses` it's better to create interfaces as you set them to state in the `App` component, so they should be the same